packages/create-drupal-decoupled/src/helpers/scaffolding.ts

```diff
@@ -0,0 +1,53 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import { SupportedFrontend } from 'src/constants'
+
+const SCAFFOLD_FILES_PER_FRONTEND = {
+  remix: {
+    files: [
+      {
+        folderPath: 'app/utils',
+        fileName: 'drupal-client.server.ts',
+      },
+      {
+        folderPath: '.',
+        fileName: 'env.example',
+        // This is due the copy plugin from esbuild, it doesn't allow to copy files with a dot at the beginning
```

Comment:
If we do a `cp -r` this won't be an issue

Reply:
You right